package.json

```diff
@@ -20,14 +20,14 @@
 	},
 	"private": true,
 	"dependencies": {
-		"@angular/animations": "~11.2.1",
-		"@angular/common": "~11.2.1",
-		"@angular/compiler": "~11.2.1",
-		"@angular/core": "~11.2.1",
-		"@angular/forms": "~11.2.1",
-		"@angular/platform-browser": "~11.2.1",
-		"@angular/platform-browser-dynamic": "~11.2.1",
-		"@angular/router": "~11.2.1",
+		"@angular/animations": "^13.0.0",
```

Comment:
This should be updated to use: `12.2.12` to match

Reply:
Updated 👍